app/javascript/dashboard/components/helpCenter/Portal/Switch/Index.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div class="portal" :class="{ active }">
+    <div class="avatar-container">
```

Comment:
There is no declaration for the class name `avatar-container`

Reply:
Fixed @muhsin-k